etl/src/main/scala/etl/RecipeParsing.scala

```diff
@@ -79,8 +80,10 @@ object RecipeParsing {
 
   def guessSteps(body: Seq[Element]): Seq[String] = {
     val candidates: Seq[Element] = body.filter(NumberedParagraph.matches)
-    candidates.map(_.text).filter(_.nonEmpty)
+    val stepsFound = candidates.map(_.text).filter(_.nonEmpty)
+    val pattern = new scala.util.matching.Regex("""(^\d+.?\s?)(.*+)""", "number", "step")
```

Comment:
Put the regex in an object as its compilaton is perf costly, see my last ophan PR